### PR TITLE
Fixes missing setting of the Window state for TD.

### PR DESCRIPTION
**Pull request by tomsons26**
Original code passed on the WinMain's command_show argument to the function.
The 8fa69511 refactor broke this, defaulting to 0, SW_HIDE, which hid the window making TD seemingly freeze and do nothing when ran.

### CODE REVIEW

File: tiberiandawn/startup.cpp [hunk 1]
```diff
@@ -313,7 +316,7 @@ int main(int argc, char** argv)
             CCDebugString("C&C95 - Creating main window.\n");
 
 #ifdef _WIN32
-            Create_Main_Window(ProgramInstance, 0, ScreenWidth, ScreenHeight);
+            Create_Main_Window(ProgramInstance, CommandShow, ScreenWidth, ScreenHeight);
```
Comment (OmniBlade) [inline tiberiandawn/startup.cpp]: No need to revert this change, you can get the needed info the same way the crt entry point does to pass to WinMain as show here https://github.com/TheAssemblyArmada/Chronoshift/blob/9fd980161bb5aae1fa7d33da699570d7e61f111e/src/game/main.cpp#L143

Comment (tomsons26) [reply tiberiandawn/startup.cpp]: Done

**Review by hifi — APPROVED** [review]
